lib/convertor.js

```diff
@@ -40,60 +40,136 @@ class MongoToKnex {
         return 'andWhere';
     }
 
-    processField(field, op) {
-        const fieldParts = field.split('.');
+    /**
+     * Determine if statement lives on parent table or if statement refers to a relation.
+     */
+    processStatement(column, op, value) {
+        const columnParts = column.split('.');
+
+        // CASE: `posts.status` -> where "posts" is the parent table
+        if (columnParts[0] === this.tableName) {
+            return {
+                column: column,
+                value: value,
+                isRelation: false
+            };
+        }
+
+        // CASE: relation?
+        if (columnParts.length > 1) {
+            debug(columnParts);
+
+            const table = columnParts[0];
+            const relation = this.config.relations[table];
+
+            if (!relation) {
+                debug('Can\'t find relation in config object.');
```

Comment:
Think this is a right way to handle it (falling back to forming non-relational query). Would there be a valid case, where we might want to throw an error here instead?

Reply:
Not sure yet 🤔 This has two potential cases:

1. Mis-usage of the user (he uses a filter which is not valid) - error?
2. Mis-Configuration from developer (he forgot to define a relation)

Reply:
And now the question is, should we throw in these situations? Think knex will throw later on, when executing the query, but maybe throwing early would make the problem more clear/discoverable? The library doesn't ever throw, so maybe we should not invent a new behavior atm. But could be good to think through some future 'debuging' scenarios :)

Reply:
we can throw for now and then see if we want to keep it?

Reply:
Ok :) Just leave a comment about it being just 'getting a feeling of it' change and we should def decide if we want to keep it when integrating with Ghost